UnityProject/Assets/Scripts/UI/UIManager.cs

```diff
@@ -107,6 +109,11 @@ public static string SetToolTip
 	/// </summary>
 	public static bool IsOxygen { get; set; }
 
+	private void Start()
+	{
+		generalSettingsMenu.Init();
```

Comment:
```
NullReferenceException: Object reference not set to an instance of an object
UIManager.Start () (at Assets/Scripts/UI/UIManager.cs:114)
```
It's not assigned

Reply:
Damn, must've forgotten to apply prefab. Sorry, should have double checked if I missed anything.

Fixed it.